samples/matter/light_bulb/overlay-aws-iot-integration.conf

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable matter integration layer
+CONFIG_AWS_IOT_INTEGRATION=y
+CONFIG_AWS_IOT_INTEGRATION_LOG_LEVEL_DBG=y
+
+# AWS IoT library
+CONFIG_AWS_IOT=y
+CONFIG_AWS_IOT_BROKER_HOST_NAME="xxx.amazonaws.com"
+CONFIG_AWS_IOT_CLIENT_ID_STATIC="my-thing"
+CONFIG_AWS_IOT_AUTO_DEVICE_SHADOW_REQUEST=n
+CONFIG_AWS_IOT_TOPIC_UPDATE_DELTA_SUBSCRIBE=y
+
+# JSON
+CONFIG_JSON_LIBRARY=y
+
+# MQTT helper library
+CONFIG_MQTT_HELPER_PROVISION_CERTIFICATES=y
+CONFIG_MQTT_HELPER_SEC_TAG=201
+
+# MQTT - Maximum MQTT keepalive timeout specified by AWS IoT Core
+CONFIG_MQTT_KEEPALIVE=1200
+CONFIG_MQTT_CLEAN_SESSION=y
+
+# MBed TLS
+CONFIG_MBEDTLS_HEAP_SIZE=98304
+CONFIG_MBEDTLS_RSA_C=y
+
+# Zephyr Networking
+CONFIG_NET_CONNECTION_MANAGER=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_TCP=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_MAX_CONN=8
+
+# Adjust NET configurations to reduce the likelyhood of congestion in the TCP stack when there are
+# multiple NET users in the system.
+CONFIG_NET_PKT_RX_COUNT=14
+CONFIG_NET_PKT_TX_COUNT=14
+CONFIG_NET_BUF_RX_COUNT=54
+CONFIG_NET_BUF_TX_COUNT=54
+CONFIG_NET_TCP_MAX_SEND_WINDOW_SIZE=1024
```

Comment:
Do we need to specify the send window? If so, it would be nice with an explanation why a window of 1024 is the maximum

Reply:
There is a comment above this block of options that states why it is needed. To you think its sufficient?

Reply:
Oh, overlooked that, it's probably fine then